lexicons/com/atproto/sync/getHead.json

```diff
@@ -25,7 +25,10 @@
             "root": {"type": "string", "format": "cid"}
           }
         }
-      }
+      },
+      "errors": [
+        {"name": "HeadNotFound"}
```

Comment:
wondering if we should take this chance to change the name of this route to `getRoot`?

the git metaphor of HEAD makes less sense if we're getting rid of history & we've gotten some.... jokes... about the current name 😅